config/prod.exs

```diff
@@ -10,8 +10,8 @@ use Mix.Config
 # which you should run after static files are built and
 # before starting your production server.
 config :apr, AprWeb.Endpoint,
-  http: [:inet6, port: System.get_env("PORT") || 5000],
-  url: [scheme: "http", host: System.get_env("APRD_HOST"), port: 80],
+  http: [:inet6, port: {:system, "PORT"} || 8080],
+  url: [scheme: "https", host: {:system, "HOST"}, port: 443],
```

Comment:
@ashkan18 might this be the cause of the 502 we were seeing when Nginx proxied to 8080?

Reply:
don't think so, as far as i understand, this is only used for redirects in the app, i may be wrong though. i can give it a try